pymc3/tests/test_smc.py

```diff
@@ -189,3 +189,8 @@ def test_repr_latex(self):
         assert expected == self.s._repr_latex_()
         assert self.s._repr_latex_() == self.s.__latex__()
         assert self.SMABC_test.model._repr_latex_() == self.SMABC_test.model.__latex__()
+
+    def test_name_is_string_type(self):
+        with self.SMABC_potential:
+            trace = pm.sample_smc(draws=10, kernel="ABC")
+            assert isinstance(trace._straces[0].name, str)
```

Comment:
could this line be added to the end of a previous test (e.g. `test_potential`)? The test suite is already pretty long

Also, it is possible to check the actual names, rather than just making assertions about the types? E.g. something like
```python
assert [i.name for i in trace._straces] == ['foo', 'bar']
```

Reply:
Thanks @MarcoGorelli for requesting this change. Turns out named models aren't actually supported. (see main comment)